test/bigtest/tests/mapping-profiles/mapping-profile-details-test.js

```diff
@@ -1709,6 +1707,82 @@ describe('Mapping Profile View', () => {
               expect(mappingProfileForm.submitFormButtonDisabled).to.be.false;
             });
           });
+
+          describe('row has subfields', () => {
+            beforeEach(async () => {
+              await mappingProfileForm.marcDetailsTable.rows(0).action.selectAndBlur('Add');
+              await mappingProfileForm.marcDetailsTable.rows(0).subaction.selectAndBlur('Add subfield');
+            });
+
+            describe('wnen "Add" action and "Add subfield" subaction are selected', () => {
+              it('then subfield row is added', () => {
+                expect(mappingProfileForm.marcDetailsTable.rows(0).subfieldsCount).to.be.equal(1);
+              });
+
+              it('subfield row do not includes up/down arrows', () => {
+                expect(mappingProfileForm.marcDetailsTable.rows(0).subfields(0).moveRowUp.isPresent).to.be.false;
+                expect(mappingProfileForm.marcDetailsTable.rows(0).subfields(0).moveRowDown.isPresent).to.be.false;
+              });
+
+              it('subfield row do not includes "position" field', () => {
+                expect(mappingProfileForm.marcDetailsTable.rows(0).subfields(0).position.hasSelect).to.be.false;
+              });
+
+              it('subfield row do not includes "add" button', () => {
+                expect(mappingProfileForm.marcDetailsTable.rows(0).subfields(0).addRow.isPresent).to.be.false;
+              });
+
+              describe('when data is filled in for main row', () => {
+                beforeEach(async () => {
+                  await mappingProfileForm.marcDetailsTable.rows(0).tag.fillAndBlur('910');
+                  await mappingProfileForm.marcDetailsTable.rows(0).indicator1.fillAndBlur('*');
+                  await mappingProfileForm.marcDetailsTable.rows(0).indicator2.fillAndBlur('*');
```

Comment:
As far as I know, asterisk characters are not allowed in the indicators fields, aren't they?

Reply:
You are right, thanks, forgot to update tests after merge validation